content/20.pharmaceuticals.md

```diff
@@ -712,6 +712,24 @@ Participants reported mild and moderate systemic adverse events after the day 1
 The second injection led to severe systemic adverse events for three of the participants at the highest dose levels, with one participant in the group being evaluated at an urgent care center on the day after the second dose.
 The reported localized adverse events from the second dose were similar to those from the first.
 
+ModernaTX was, in fact, the second company to release news of a successful interim analysis of an mRNA vaccine.
+The first report came from Pfizer and BioNTech's mRNA vaccine BNT162b2 on November 9, 2020 [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-vaccine-candidate-against].
+A preliminary report was published in the _New England Journal of Medicine_ one month later [@doi:10.1056/NEJMoa2034577].
+The vaccine candidate is comprised of the full SARS-CoV-2 spike protein in a vaccine formulation based on modRNA technology [@doi:10.1101/cshperspect.a014035].
+During the Phase III trial, 43,538 participants were enrolled 1:1 in the placebo and the vaccine candidate and received two 30-μg doses 21 days apart [@doi:10.1056/NEJMoa2034577].
+Of these enrolled participants, 21,720 received BNT162b2 and 21,728 received a placebo [@doi:10.1056/NEJMoa2034577].
+Recruitment occurred at 135 sites across six countries: Argentina, Brazil, Germany, South Africa, Turkey, and the United States.
+An initial press release described the first 94 cases, which were consistent with 90% efficacy of the vaccine at 7 days following the second dose [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-vaccine-candidate-against].
+The release of the full trial information included additional cases and analyzed the first 170 cases occurring at least 7 days after the second dose, 8 of which occurred in patients who had received BNT162b2.
+The first interim analysis reported the results of the first 94 cases of COVID-19 occurring at least 7 days after the second dose.
+Of these, 8 occurred in participants who received BNT162b2 and 162 occurred
```

Comment:
Is this sentence unfinished?

Reply:
Thanks for catching this! I think I failed to delete the original two lines from before the new data was released.